utils/walker_visitor.go

```diff
@@ -1,13 +0,0 @@
-package utils
-
-// Walker is a portion of the config that can be walked.
-type Walker interface {
```

Comment:
the walker interface is only used right now to replace placeholders, so thought it may make sense to remove it because the code is fragile (uses a lot of reflections) and complex. 

If we think we want to use this to modify other things I can just remove the references for placeholders and keep the actual interface in.